e2e/cypress/integration/messaging/emoji_size_spec.js

```diff
@@ -84,4 +84,26 @@ describe('Messaging', () => {
         // # Making sure Emojis from last post message are 21px size
         checkEmojiSize('@newLineMessage', emojis, false);
     });
+
+    it('MM-17457 Unicode and emoticon', () => {
```

Comment:
```suggestion
    it('M17457 Emojis show as jumbo in main thread - Multi emoji, no text, including unicode and emoticon', () => {
```

Reply:
Was `M17457` a typo? (and intended to be `MM-14757`)

Reply:
it was sorry, I just push a fix for this.

Reply:
@jasonblais That's correct.  It's the test key we're using to identify test case in our release spreadsheet. `M17457` means "M" as Messaging tab in release spreadsheet with "17457" as MM-17457 Jira ticket.  We documented this at - https://developers.mattermost.com/contribute/webapp/end-to-end-tests/#writing-specs